### DISCUSSION TITLE
Would there be any downsides to using an async engine for sqlachlemy core / orm and a sync engine for alembic?

### DISCUSSION THREAD

**Discussion by aaronsgithub** [sqlalchemy/alembic#1319] [Usage Questions] [answered]
Other than having to install a sync and async version of the database connectors I can't think of any.  
I just wanted to check if I'm missing anything.

**Comment by CaselIT** [comment, marked as answer]
Hi,

Probably not, but unless you have a particular reason to do so, you are probably better off using the async engine also with alembic
